core/App/components/misc/CredentialCard.tsx

```diff
@@ -162,8 +160,6 @@ const CredentialCard: React.FC<CredentialCardProps> = ({ credential, style = {},
       if (!indyCredentialFormat) {
         return
       }
-      const isRevoked = revoked.has(indyCredentialFormat.credentialRecordId)
```

Comment:
Since this component is used across different views, I wanted to avoid having to pass the revocation state as a prop since it can be pulled from the credential record. Will that no longer work with the revocation messaging?

Reply:
Oh yeah, we're passing the whole credential so it should be fairly easy to do it that way instead. I'll update that while I fix the remaining linting errors.